### PR TITLE
5 combine points to places

### PR DESCRIPTION
**Pull request by Kell-Stack**
created two migrations. converted points and place into place. updated place.rb with new coulmn information.

### CODE REVIEW

File: rails/db/migrate/20190331181118_convert_point_to_place_id.rb [hunk 1]
```diff
@@ -0,0 +1,11 @@
+class ConvertPointToPlaceId < ActiveRecord::Migration[5.2]
+  def change
+    points = Point.all.index_by(&:place_id)
+      Place.find_each do |place|
+        point = points[place.id]
+        place.update!(
+          lat: point.lat,
+          lng: point.lng
+        )
```
Comment (cmhnk) [inline rails/db/migrate/20190331181118_convert_point_to_place_id.rb]: I think you're missing an `end` here to close the block started on line 4

File: rails/app/models/place.rb [hunk 2]
```diff
@@ -5,7 +5,8 @@ class Place < ApplicationRecord
   def self.import_csv(filename)
     CSV.parse(filename, headers: true) do |row|
       loc = Place.where(name: row[0], type_of_place: row[1]).first_or_create
-      loc.points.create(title:row[0], lat: row[5].to_f, lng: row[4].to_f, region: row[3] )
+      loc.points.create(title:row[0], lat: row[5].to_f, lng: row[4].to_f,
+      region: row[3] )
```
Comment (baweaver) [inline rails/app/models/place.rb]: Nit: We might consider putting these back on one line and taking out the space after `row[3]`

File: rails/db/migrate/20190331181118_convert_point_to_place_id.rb (hunk 1 above)
Comment (baweaver) [reply rails/db/migrate/20190331181118_convert_point_to_place_id.rb]: This section is also indented an extra level

**Comment by Kell-Stack** [comment]
once PR is approved will work on drop points table migration

File: rails/db/migrate/20190331181118_convert_point_to_place_id.rb [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+        Place.find_each do |place|
+          point = points[place.id]
+            place.update!(
+              lat: point.lat,
+              lng: point.lng
+            )
+        end
+
+  end
+end
```
Comment (baweaver) [inline rails/db/migrate/20190331181118_convert_point_to_place_id.rb]: Nit: indentation is still off a bit:

```ruby
class ConvertPointToPlaceId < ActiveRecord::Migration[5.2]
  def change
    points = Point.all.index_by(&:place_id)
    
    Place.find_each do |place|
      point = points[place.id]

      place.update!(
        lat: point.lat,
        lng: point.lng
      )
    end
  end
end
```

**Pull request closed without merging**